src/Nox.Ui.Blazor.Lib/Components/NoxTypes/EditMonth.razor.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.AspNetCore.Components;
+using Nox.Types;
+using Nox.Ui.Blazor.Lib.Models;
+using System.Globalization;
+
+namespace Nox.Ui.Blazor.Lib.Components.NoxTypes;
+
+public partial class EditMonth : ComponentBase
+{
+
+    #region Declarations
+
+    [Parameter]
+    public byte? Month { get; set; }
+
+    [Parameter]
+    public CultureInfo CultureInfo { get; set; } = CultureInfo.CurrentCulture;
+
+    [Parameter]
+    public string? Title { get; set; }
+
+    [Parameter]
+    public EventCallback<byte?> MonthChanged { get; set; }
+
+    public Dictionary<byte, string?>? MonthSelectionList { get; set; } = new Dictionary<byte, string?>();
+
+    public string? CurrentMonthStr { get; set; }
+
+    #endregion
+
+    /// <summary>
+    /// Handles initial loading
+    /// </summary>
+    protected override void OnInitialized()
+    {
+        var months = Enumerable.Range(1, 12).Select(i => new { I = i, M = CultureInfo.GetCultureInfo(CultureInfo.LCID).DateTimeFormat?.GetMonthName(i) });
+
+        foreach (var CurrentMonth in months)
+        {
+            MonthSelectionList?.Add((byte)CurrentMonth.I, CurrentMonth.M);
```

Comment:
the null check is unnecessary?
```suggestion
            MonthSelectionList.Add((byte)CurrentMonth.I, CurrentMonth.M);
```

Reply:
resolved on commit